contracts/src/core/AlignedLayerServiceManager.sol

```diff
@@ -148,6 +157,7 @@ contract AlignedLayerServiceManager is
             batchesState[batchMerkleRoot].batcherAddress
         ] -= txCost;
```

Comment:
Should emit BatcherBalanceUpdated.

Also in `receive()` , but i can't leave a comment on that line .-.

Reply:
Solved in [e4e648b](https://github.com/yetanotherco/aligned_layer/pull/840/commits/e4e648bb1b9e00c20215d4562837c516abbb2142)